.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import { addDecorator, addParameters } from '@storybook/html';
+import { sanitize } from '@storybook/csf';
```

Comment:
Shouldn't this dependency be added to `package.json`?

Reply:
It appears it was included by default with Storybook, but I can try to make it a separate dependency.

Reply:
Because it's a namespaced dependency, it would be good to track it in `package.json` just in case Storybook ever moves away from a monorepo strategy (and for consistency).

Reply:
Added